sw/device/silicon_creator/lib/ownership/isfb.c

```diff
@@ -0,0 +1,120 @@
+// Copyright lowRISC contributors (OpenTitan project).
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/silicon_creator/lib/drivers/flash_ctrl.h"
+#include "sw/device/silicon_creator/lib/error.h"
+#include "sw/device/silicon_creator/lib/manifest.h"
+#include "sw/device/silicon_creator/lib/ownership/owner_block.h"
+
+enum {
+  kFlashWordSize = 64,
+
+  // The number of bits in the strike mask is fixed to 128. Each bit in the
+  // strike mask corresponds to a flash word.
+  kExpectedStrikeBitCount = 128,
+
+  // The expected size of the strike region in bytes and words.
+  kExpectedStrikeRegionBytesCount =
+      (kExpectedStrikeBitCount * kFlashWordSize) / CHAR_BIT,
+  kExpectedStrikeRegionWordsCount =
+      kExpectedStrikeRegionBytesCount / sizeof(uint32_t),
+};
+
+rom_error_t isfb_boot_request_process(const manifest_ext_isfb_t *ext,
+                                      const owner_config_t *owner_config,
+                                      uint32_t *checks_performed_count) {
+  *checks_performed_count = UINT32_MAX;
+  if (ext->header.name == kManifestExtIdIsfb &&
```

Comment:
what other values `ext->header.name` is allowed to have here?

shouldn't the check be

`(ext->header.name != kManifestExtIdIsfb || (hardened_bool_t)owner_config->isfb == kHardenedBoolFalse)` ?

Reply:
> what other values ext->header.name is allowed to have here?

All manifest extensions use the same header. The previous check was incorrect. Updated to match your recommendation. 

Done